src/MagnumPlugins/ObjImporter/Test/Test.cpp

```diff
@@ -311,30 +333,25 @@ void ObjImporterTest::textureCoordinatesNormals() {
         {0.5f, 1.0f, 0.5f}
     }));
     CORRADE_COMPARE(data->indices(), (std::vector<UnsignedInt>{
-        0, 1, 2, 3, 1, 0, 4, 2
+        0, 1, 2, 3, 1, 0, 4, 2, 2
     }));
 }
 
 void ObjImporterTest::emptyFile() {
     ObjImporter importer;
     CORRADE_VERIFY(importer.openFile(Utility::Directory::join(OBJIMPORTER_TEST_DIR, "emptyFile.obj")));
-    CORRADE_COMPARE(importer.mesh3DCount(), 1);
-}
-
-void ObjImporterTest::unnamedMesh() {
```

Comment:
Why is this removed? Then there's no test case that tests that `mesh3DForName("")` returns -1